runtime/src/lib.rs

```diff
@@ -1796,12 +1789,16 @@ impl_runtime_apis! {
     }
 
     impl sp_genesis_builder::GenesisBuilder<Block> for Runtime {
-        fn create_default_config() -> Vec<u8> {
-            create_default_config::<RuntimeGenesisConfig>()
+        fn build_state(config: Vec<u8>) -> sp_genesis_builder::Result {
+            build_state::<RuntimeGenesisConfig>(config)
+        }
+
+        fn get_preset(id: &Option<sp_genesis_builder::PresetId>) -> Option<Vec<u8>> {
+            get_preset::<RuntimeGenesisConfig>(id, |_| None)
         }
 
-        fn build_config(config: Vec<u8>) -> sp_genesis_builder::Result {
-            build_config::<RuntimeGenesisConfig>(config)
+        fn preset_names() -> Vec<sp_genesis_builder::PresetId> {
+            vec![]
```

Comment:
This would be a list of ids of genesis configurations supported by this runtime.

This is part of a mechanism which completely decouples the runtime from the node, and allows new functionalities (e.g. see the `chain-spec-builder` tool).

I would opt out of this for now (and thus leave it empty), given the bigger refactoring needed, and the limited number of configurations/chains that we support.

You can also refer to https://github.com/paritytech/polkadot-sdk/pull/2714 for more info.